pkg/controller/elasticsearch/validation/validations.go

```diff
@@ -272,11 +272,11 @@ func noDowngrades(current, proposed esv1.Elasticsearch) field.ErrorList {
 
 func validUpgradePath(current, proposed esv1.Elasticsearch) field.ErrorList {
 	var errs field.ErrorList
-	currentVer, err := version.Parse(current.Spec.Version)
-	if err != nil {
-		// this should not happen, since this is the already persisted version
-		errs = append(errs, field.Invalid(field.NewPath("spec").Child("version"), current.Spec.Version, parseStoredVersionErrMsg))
+	currentVer, ferr := currentVersion(current)
+	if ferr != nil {
```

Comment:
`ferr`?

Reply:
`fear` for `field.Error` which is what the webhook API works with, `err` is used as a variable for a regular builtin `error` below